Core/scripts/dirac-install.py

```diff
@@ -2047,6 +2048,12 @@ def createPermanentDirLinks():
                 os.makedirs(os.path.join(real, fd))
           os.rename(fake, fake + '.bak')
         os.symlink(real, fake)
+
+      # Files
+      for filename in ['bashrc']:
```

Comment:
Why a loop here?

Reply:
I think this is by analogy with the directories, which is fine